docs/src/running-validator/vote-accounts.md

```diff
@@ -0,0 +1,117 @@
+
+### Vote Authority
+
+The _vote authority_ keypair is used to sign each vote transaction the validator
+node wants to submit to the cluster.  This doesn't necessarily have to be unique
+from the validator identity, as you will see later in this document.  Because
+the vote authority, like the validator identity, is signing transactions
+frequently, this also must be a hot keypair on the same file system as the
+validator process.
+
+It is recommended to set the vote authority to the same address as the validator
+identity.  If the validator identity is also the vote authority, only one
+signature per vote transaction is needed in order to both sign the vote and pay
+the transaction fee.  Because transaction fees on Solana are assessed
+per-signature, having one signer instead of two will result in half the transaction
+fee paid compared to setting the vote authority and validator identity to two
+different accounts.
+
+The vote authority can be set when the vote account is created.  If it is not
+provided, the default behavior is to assign it the same as the validator identity.
+The vote authority can be changed later with the `vote-authorize-voter` command.
+
+The vote authority can be changed at most once per epoch.  If the authority is
+changed with `vote-authorize-voter`, this will not take effect until the
+beginning of the next epoch.  To support a smooth transition of the vote signing,
+`solana-validator` allows the `--authorized-voter` argument to be specified
+multiple times.  This allows the validator process to keep voting successfully
+when the network reaches an epoch boundary at with the validator's vote authority
+account changes.
+
+### Withdraw Authority
+
+The _withdraw authority_ keypair is used to withdraw funds from a vote account
+using the `withdraw-from-vote-account` command.  Any network rewards a validator
+earns are deposited into the vote account and are only retrievable by signing
+with the withdraw authority keypair.
+
+The withdraw authority is also required to sign any transaction to change
+a vote account's [commission](#commission), and to change the validator
+identity on a vote account.
+
+Because the vote account could accrue a significant balance, it is recommended
+to keep the withdraw authority keypair in an offline/cold wallet, as it is
+not needed to sign frequent transactions.
+
+The withdraw authority can be set at vote account creation with the
+`--authorized-withdrawer` option.  If this is not provided, the validator
+identity will be set as the withdraw authority by default.
+
+The withdraw authority can be changed later with the `vote-authorize-withdrawer`
+command.
+
+### Commission
+
+_Commission_ is the percent of network rewards earned by a validator that are
+deposited into the validator's vote account.  The remainder of the rewards
+are distributed to all of the stake accounts delegated to that vote account,
+proportional to the active stake weight of each stake account.
```

Comment:
How about another sentence or two explaining why a validator might set a commission to 0%, 50% or 100%?